### PR TITLE
Reformat Format-Table tests

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

`Format-Table` tests have gaps and any changes to the formatting subsystem have a risk of regressing.
Before adding new tests we have to reformat the file - replace indentations with tabs, remove aliases (we have separate tests for aliases) and extra line, remove extra `Describe`.

Each commit contains separate change.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by daxian-dbw — APPROVED** [review]
LGTM.
@SteveL-MSFT and @adityapatwardhan, the diff is much cleaner if you review this PR in code flow.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 1]
```diff
@@ -1,258 +1,247 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Describe "Format-Table" -Tags "CI" {
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Can you insert whitespace below the copyright?

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 2]
```diff
@@ -1,258 +1,247 @@
...
-		}
-
-		It "Format-Table should have correct alignment" {
-			$ps1xml = @"
+        It "Should call format table on piped input without error" {
+                { Get-Date | Format-Table } | Should -Not -Throw
+        }
+
+        It "Should return a format object data type" {
+                $val = (Get-Date | Format-Table | Get-Member )
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Add space between open parenthesis and `Get-Date`

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 3]
```diff
@@ -1,258 +1,247 @@
...
+                $val2 = (Get-Date | Format-Table | Get-Member )
+
+                $val.TypeName | Should -Match "Microsoft.Powershell.Commands.Internal.Format"
+                $val2.TypeName | Should -Match "Microsoft.Powershell.Commands.Internal.Format"
+        }
+
+        It "Should be able to be called with optional parameters" {
+                $v1 = (Get-Date | Format-Table *)
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Add space between open parenthesis and `Get-Date`

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 4]
```diff
@@ -1,258 +1,247 @@
...
+                $e = { $obj | Format-Table -view bar -force -EA Stop } |
+                    Should -Throw -ErrorId "FormatViewNotFound,Microsoft.PowerShell.Commands.FormatTableCommand" -PassThru
+                $e.CategoryInfo | Should -Match "PipelineStoppedException"
+        }
+
+        It "Format-Table with array should work" {
+                $al = (0..255)
+                $info = @{}
+                $info.array = $al
+                $result = $info|Format-Table|Out-String
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Add whitespace around pipe symbols, same below for `$result|Out-String`

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 2 above)
Comment (daxian-dbw) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Actually, the parenthesis seems unnecessary, can we remove it?

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 3 above)
Comment (daxian-dbw) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Same here. Maybe remove the parenthesis?

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 1 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: @SteveL-MSFT Seem we lost this in all test files. 😕 I believe we should fix this all in new PR.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 2 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Removed all extra parentheses.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 3 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Removed.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 4 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Fixed. Below too.

**Review by daxian-dbw — APPROVED** [review]

**Comment by iSazonov** [comment]
@SteveL-MSFT Could you please continue?

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**